conf/ipa.yaml.template

```diff
@@ -15,4 +15,4 @@ IPA:
   GROUPS:
     - satadmins
     - satusers
-  KEYTAB:
+  KEYTAB_URL:
```

Comment:
is there an MR made for this change in a satelliteqe-jenkins repo?

Reply:
yes, !451

Reply:
Thanks, I see, MR!451 is pending due to satlab-tower MR!411, can you move this PR to DRAFT?

Reply:
Don't hesitate to approve it as well - makes for a faster merge coming out of draft if no further commits are pushed after the approval.

Reply:
all related MRs are ready for review ;)